cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -184,9 +184,6 @@ def test_verify_revert_snapshot(full_test_vm, provider, soft_assert, register_ev
     full_test_vm.wait_for_vm_state_change(desired_state=full_test_vm.STATE_OFF, timeout=720)
     full_test_vm.power_control_from_cfme(option=full_test_vm.POWER_ON, cancel=False)
     full_test_vm.wait_for_vm_state_change(desired_state=full_test_vm.STATE_ON, timeout=900)
-    current_state = full_test_vm.find_quadicon().data['state']
-    soft_assert(current_state.startswith('currentstate-on'),
```

Comment:
**Question**: Shouldn't we check if the vm state is really changed?

Reply:
wait_for_vm_state_change() does the checking. If the state is anything other than what we expect, the function fails at wait_for(). It makes no sense to have the soft asserts here, since it will always be True, and if wait_for_vm_state_change() fails, it will never get to them.